### DISCUSSION TITLE
Welcome, Veer Pratap!

### DISCUSSION THREAD

**Discussion by Veercodeprog** [ParanoidUser/yolo#537] [Subscribe]
would love to contribute

**Comment by mayankjaiswal1709** [comment]
yes

**Reply by Veercodeprog** [reply]
i am creating a PR ,feel free to co-author

**Comment by ParanoidUser** [comment]
Ahoy @Veercodeprog, welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers

**Comment by Veercodeprog** [comment]
not sure how pair-ordinaire works?

**Reply by ParanoidUser** [reply]
Here your are 😉 The trick was to use the actual combination of username & email.
https://github.com/Veercodeprog?tab=achievements&achievement=pair-extraordinaire
